worlds/faxanadu/docs/setup_en.md

```diff
@@ -0,0 +1,32 @@
+# Faxanadu Randomizer Setup
+
+## Required Software
+
+- [Daxanadu](https://github.com/Daivuk/Daxanadu/releases/)
+- Faxanadu ROM, English version
+
+## Optional Software
+
+- [ArchipelagoTextClient](https://github.com/ArchipelagoMW/Archipelago/releases)
+
+## Installing Daxanadu
+1. Download [Daxanadu.zip](https://github.com/Daivuk/Daxanadu/releases/) and extract it.
+2. Copy your rom `Faxanadu (U).nes` into the newly extracted folder.
+
+## Joining a MultiWorld Game
+
+1. Launch Daxanadu.exe
+2. From the Main menu, Go to `ARCHIPELAGO` menu. Enter Address, slot name and password. Then select `PLAY`.
+3. Enjoy!
+
+To continue a game, follow the same connection steps.
+Connecting with a different seed won't erase your progress in other seeds.
```

Comment:
does the client have a way to edit the port if it has changed without creating a new save?

Reply:
The save filename is a combination of seed + hashed username.
They can change the port at anytime during connection process:
![image](https://github.com/ArchipelagoMW/Archipelago/assets/2482827/d21e5812-3473-484d-9ecc-5528cf2ff4fb)